File: DSA/Assignment_3/diagonal_sum.c
#include <stdio.h>
int main()
{
    int m,n;
    printf("Enter the row and column size:");
    scnaf("%d %d",&m,&n);
    int arr[m][n];
    for(int i=0;i<m;i++)
    {
        for(int j=0;j<n;j++)
        {
            printf("Enter element:");
            scanf("%d",arr[i][j]);
        }
    }
    if(m==n)
    {}
    
    return 0;
}

File: DSA/Assignment_3/diagonal_sum.py
m=int(input("Enter row size:"))
n=int(input("Enter column size:"))
arr=[]
sum=0
for i in range(m):
    a=[]
    for j in range(n):
        a.append(int(input("Enter element:")))

for i in range(m):
    for j in range(n):
        if i==j:
            sum+=arr[i][j]

print("Sum of diagonal elements:",sum)